hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneCluster.java

```diff
@@ -315,6 +315,7 @@ void restartHddsDatanode(DatanodeDetails dn, boolean waitForDatanode)
     protected Optional<String> omId = Optional.empty();
     
     protected Boolean randomContainerPort = true;
+    protected Optional<String> datanodeReservedSpace = Optional.of("0B");
```

Comment:
I think a better default value would be:

```suggestion
    protected Optional<String> datanodeReservedSpace = Optional.empty();
```